youtube_dl/extractor/cba.py

```diff
@@ -0,0 +1,104 @@
+        }
+    }
+    _FORMATS = {
+        'audio/ogg': {'id': '1', 'ext': 'ogg', 'preference': 100},
+        'audio/mpeg': {'id': '2', 'ext': 'mp3', 'preference': 50}
+    }
+    _API_KEY = None
+
+    def __init__(self, *args, **kwargs):
+        try:
+            self._API_KEY = os.environ["CBA_API_KEY"]
+        except KeyError:
+            pass
+
+    def _add_optional_parameter(self, formats, name, data, key, convert=None):
+        try:
+            param = data[key]
+            if convert:
+                param = convert(param)
+            formats[name] = param
+        except KeyError:
+            pass
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        api_posts_url = "https://cba.fro.at/wp-json/wp/v2/posts/%s" % video_id
+        api_media_url = "https://cba.fro.at/wp-json/wp/v2/media?media_type=audio&parent=%s" % video_id
+
+        title = 'unknown'
+        description = ''
+        formats = []
+
+        posts_result = self._download_json(api_posts_url, video_id, 'query posts api-endpoint', 'unable to query posts api-endpoint')
+        try:
+            title = clean_html(posts_result['title']['rendered'])
```

Comment:
Yes. Though you may provide fallback sources for it (if any). But overall title should be fatal. `unknown` hardcode may be used if absence of title is allowed by source provider or in case there is no title is expected at all (in this case video id is usually used).